services/gateway/tests/functional/registerHttpApi.test.js

```diff
@@ -0,0 +1,271 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+	mapParamWithType,
+	transformParams,
+	transformRequest,
+	transformResponse,
+	registerApi,
+	transformPath,
+} = require('../../shared/registerHttpApi');
+const {
+	expectedResponseForRegisterApi,
+	methodDefForTransformResponse,
+	dataForTransformResponse,
+	expectedResponseForTransformResponse,
+	paramsForTransformRequest,
+	methodDefForTransformRequest,
+	expectedResponseForTransformRequest,
+	sourceForMapParamWithType,
+	paramsForTransformParams,
+	specsForTransformParams,
+	expectedResponseForTransformParams,
+} = require('../constants/registerApi');
+
+describe('Test transformPath method', () => {
```

Comment:
Please refactor the necessary tests according to the changes requested above (`apiUtils.js`).

Reply:
Also, kindly move these tests under `unit` tests, since they're not dependent on external entities.